### ISSUE TITLE
Add layout elements

### ISSUE THREAD

**Issue by mcdonnelldean** [nearform/vidi-dashboard#108]
I've done a first pass at the layout for the /processes screen.

__Layout grouping to support react keys__
Need a way to group rows, the issue is when you have a bunch of markdown that you use for listing content you need to wrap them in a keyed div. This means even though on screen the data looks like a number of flat rows it actually needs to be laid out like,
```html
<div className="row">..</div>
<div className="row">..</div>
<div key="1234" className="row">
  <div className="row">..</div>
  <div className="row">..</div>
</div>
<div className="row">..</div>
```
This 'group' basically needs to be a row and a container at the same time so that you can put rows inside it and it can be a row itself. This works with using `container-fluid` but it adds padding.

__Compact UI__
The UI below represents a single process, there could be 30 process on screen, we need to present the same data in a much tighter fashion but without being so tight that it's too hard to read. Most of the padded space in the image below is bad html on my part, removing this would fix most of the issue.

__More to Add__
There is more content to add in later iterations so we should have enough markup to keep adding sections.

__Listing Process__
We need enough UI to do a collapsed version of each process. Beside each process should be a collapse button. When this button is toggled we will render the full detail vs the collapsed version. For now just adding a static template at the top of the page is fine. I can use it to add the logic. The idea here is the screen will start as a list of summaries and you can expand the ones you are interested in.

<img width="843" alt="screen shot 2016-02-06 at 01 31 44" src="https://cloud.githubusercontent.com/assets/3625961/12875377/b9c6a760-cde0-11e5-842d-1dc61a2a09a6.png">

**Comment by thefoxis** [comment]
With the context that I have and UI elements currently pleasant I've made some changes and created a mockup of collapsed view. Couple remarks:

* the elements are a little bit more tight, responsive and also separated by lines
* the labels were moved and styled so they have less importance
* processes are "cards" for visual separation
* cleaned up the markup
* it'd be great to attach a class (now hardcoded) to `over limit` stat so it can be either green or red (passing or exceeded). for the sake of clarity and getting this information easily maybe it'd be good to add a little dot next to process name-pid with the same colour-coding
* collapsed view doesn't have more than uptime, but I think in reality it might need more. the question i'd have is which parts are crucial (it might be hard to reshuffle some w/o duplicating content)

let me know what you think! :rocket:
